src/dbc.rs

```diff
@@ -7,22 +7,45 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 use crate::{
-    DbcContent, DbcContentHash, DbcTransaction, Error, Hash, KeyManager, PublicKey, Result,
-    Signature,
+    DbcContent, DbcTransaction, Error, KeyManager, PublicKey, Result, Signature, SpendKey,
 };
+
 use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
+use tiny_keccak::{Hasher, Sha3};
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
 pub struct Dbc {
     pub content: DbcContent,
     pub transaction: DbcTransaction,
-    pub transaction_sigs: BTreeMap<DbcContentHash, (PublicKey, Signature)>,
+    pub transaction_sigs: BTreeMap<SpendKey, (PublicKey, Signature)>,
 }
 
 impl Dbc {
-    pub fn name(&self) -> Hash {
-        self.content.hash()
+    pub fn spend_key(&self) -> SpendKey {
```

Comment:
We should doc-comment all public APIs.

Reply:
Yes we should, I'll add a few here for now but we should add the `# deny(missing docs)` lint once the library is stable.